artifacttemplates/http%3A%2F%2Fopentosca.org%2Fartifacttemplates/DockerEngine_DockerEngine-Interface-w1/source/src/main/java/org/opentosca/artifacttemplates/Constants.java

```diff
@@ -0,0 +1,19 @@
+package org.opentosca.artifacttemplates;
+
+public class Constants {
+
+    // namespace under which the SOAP service operates
+    protected static final String NAMESPACE_URI = "http://NodeTypes.opentosca.org";
+
+    // port type to use for the SOAP service
+    protected static final String PORT_TYPE_NAME = "org_opentosca_NodeTypes_DockerEngine__InterfaceDockerEnginePort";
```

Comment:
I guess we should also adapt this name here. What about `org_opentosca_artifactTemplates_dockerEngine_DockerEngineInterfacePort`

Reply:
Sure, it just has to match `<ServiceEndpoint>/org_opentosca_NodeTypes_DockerEngine__InterfaceDockerEnginePort</ServiceEndpoint>` in the ArtifacTemplate.tosca